### ISSUE TITLE
add SSL (get a cert for testdouble.com)

### ISSUE THREAD

**Issue by searls** [testdouble/present#39]

**Comment by searls** [comment]
Hey @danthompson is this an issue that interests you? I actually bought an SSL cert before realizing heroku charges $20/month to host it — thoughts?

**Comment by danthompson** [comment]
Yeah, I dig SSL. :smile:

**Comment by searls** [comment]
Okay, fun update.

**tl;dr all traffic is forced over ssl now and it's doing it the zero-maintenance way by just funneling traffic to herokuapp.com**

Some background:

- [x] Buy an SSL cert from DNSimple 6-8 months ago without realizing there were still a bunch of config steps [link](https://devcenter.heroku.com/articles/ssl-certificate-dnsimple)
- [x] Hesitate before just adding the $20/month heroku ssl:endpoint addon, because umpteen Twitter mentions suggested there were still a lot of manual steps and ongoing security updates/maintenance to deal with [link](https://devcenter.heroku.com/articles/ssl-endpoint)
- [x] Have some folks recommend cloudflare, but that'd require we move off DNSimple b/c their integration doesn't work with ALIAS records [link](https://twitter.com/aeden/status/593160927345844224)
- [x] have other folks recommend expeditedSsl, which seems really cool, but wasn't clear that it actually solves the endpoint problem (so $15/mo for a separate cert and possible DNS issues or $35/mo all told after configuring it?) [link](https://www.expeditedssl.com)
- [x] notice in the docs that https is already secure, well-maintained, and configured for *.herokuapp.com
- [x] switch our ALIAS & TXT records to a URL record pointing folks to the app on herokuapp.com over https
- [x] force SSL in production via rails config https://github.com/testdouble/present/commit/d35a50bedd07c429160174b8ac802abc87bdd674
- [x] update the production github oauth app to redirect to the callback url on herokuapp.com
